Modules/_ssl.c

```diff
@@ -138,10 +138,10 @@ struct py_ssl_library_code {
 #else /* OpenSSL < 1.1.0 */
 #if defined(WITH_THREAD)
 #define HAVE_OPENSSL_CRYPTO_LOCK
-/* For some reason, this function is not declared on OpenSSL's headers */
-void OPENSSL_cpuid_setup(void);
 #endif
 
+/* For some reason, this function is not declared on OpenSSL's headers */
+void OPENSSL_cpuid_setup(void);
```

Comment:
Is there a nice way to verify that? I see that you have a [project to compile a set of libressl and openssl versions](https://github.com/tiran/multissl).

Checking on OpenSSL, I notice that the [commit that implemented this function is dated back to 2004](https://github.com/openssl/openssl/commit/14e21f863a3e3278bb8660ea9844e92e52e1f2f7) when 0.9.8 version didn't exist.

And on Libressl, it was [added on 2008](https://github.com/libressl/libressl/commit/a1c999dc7d25dcdde0bc38e0fe096a14c8c13b89) and 

Would that be enough?